code/bit_manipulation/invert_bit/InvertBit.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <bitset>
+
+using namespace std;
```

Comment:
Recommend not using this.

Reply:
Using directive can't use only in header file. 
```
// sample2.cpp
#include "sample2.hpp"
#include <utility>
using namespace std;    // ok, users will include .hpp not .cpp
```

Reply:
Like I said, just a recommendation, since using namespace std is considered bad practice in general.